js/coinsbit.js

```diff
@@ -0,0 +1,799 @@
+            const type = 'spot';
+            result.push ({
+                'id': id,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'type': type,
+                'spot': true,
+                'derivative': undefined,
+                'contract': undefined,
+                'linear': undefined,
+                'inverse': undefined,
+                'contractSize': undefined,
+                'expiry': undefined,
+                'expiryDatetime': undefined,
+                'strike': undefined,
+                'optionType': undefined,
+                'settle': undefined,
+                'settleId': undefined,
+                'margin': undefined,
+                'active': undefined,
+                'taker': undefined,
+                'maker': undefined,
+                'precision': {
+                    'price': pricePrecision,
+                    'amount': quantityPrecision,
+                },
+                'limits': {
+                    'leverage': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'amount': {
+                        'min': minAmount,
+                        'max': undefined,
+                    },
+                    'price': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'cost': {
+                        'min': this.safeNumber (market, 'feePrec'),     // <<< TODO >>> I am not yet sure this is correct.
+                        'max': undefined,
+                    },
+                },
+                'info': market,
+            });
+        }
+        return result;
+    }
+
+    async fetchTicker (symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        let ticker = undefined;
+        const response = await this.publicGetTicker (request);
+        //     {
+        //          success: true,
+        //          message: "",
+        //          result: {
+        //            bid: '46504.7412435',
+        //            ask: '46511.54203562',
+        //            open: '47030.33',
+        //            high: '47549.99',
+        //            low: '45571.08',
+        //            last: '46459.99',
+        //            volume: '8372.29022355',
+        //            deal: '388090406.73607745',
+        //          },
+        //          code: "200"
+        //     }
+        ticker = this.safeValue (response, 'result', {});
+        return this.parseTicker (ticker, market);
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        const request = params;
```

Comment:
Let's pass the `params` directly to the endpoint, we don't need the `request` variable here.